lib/puppet/type/f5_command.rb

```diff
@@ -18,7 +18,7 @@
 
   newproperty(:description, :parent => Puppet::Property::F5Description)
 
-  newparam(:tmsh) do
+  newproperty(:tmsh) do
```

Comment:
A property is something you should be able to get the state of.  It was more correct before as a param.